### ISSUE TITLE
eventStore event handler is not called

### ISSUE THREAD

**Issue by artemjackson** [reimagined/resolve#957]
**Describe the bug**

`eventStore` event handler is not called if `eventFilter.eventTypes` and `eventFilter.aggregateIds` is not defined.

**Steps to reproduce**

```js
// import any createStoreAdapter
// import any createBusAdapter

const eventStore = createEventStore({
  storage: createStorageAdapter(storageOpts),
  bus: createBusAdapter(busOpts),
});

eventStore.loadEvents({ skipStorage: true }, (event) => {
  console.log('Received event:', event); // never called :\
});

eventStore.saveEvent({
  aggregateId: 1,
  type: 'demo-event',
  ...otherEventOpts
});
```

**Actual behavior**
The handler is not called

**Expected behavior**
The handler is called

**Possible issue fix**

I noticed that each `resolve-bus-<mq>` package has `onMessage` with one [confusing me line](https://github.com/reimagined/resolve/blob/master/packages/adapters/bus-adapters/resolve-bus-rabbitmq/src/on-message.js#L19):

```js
for (const topic of topics) {
      const topicHandlers = pool.handlers.get(topic)
      if (topicHandlers == null) return // Should there actually be return ?

      for (const handler of Array.from(topicHandlers)) {
        applicationPromises.push(handler(event))
      }
    }
```

The code above stops on first(!) not matched topic which is probably weird.
I think that there should be `continue` instead of `return`.

Am I right?

Should I provide a PR with fix?

**Comment by IhostVlad** [comment]
@artemjackson Thanks for reporting issue. We will fix it in next library version (0.19.1).

> I think that there should be continue instead of return. Am I right?
Yes, you are right

**Comment by IhostVlad** [comment]
Done in 0.19.1
